test/test_supervisor.rb

```diff
@@ -1054,6 +1054,47 @@ def test_stop_parallel_old_supervisor_after_delay
     end
   end
 
+  sub_test_case "include additional configuration" do
+    setup do
+      omit "Not supported on Windows" if Fluent.windows?
```

Comment:
> omit "Not supported on Windows" if Fluent.windows?

Is this feature not supported on Windows?

Reply:
Not limited.
Initially skipped.